.gitignore

```diff
@@ -0,0 +1,13 @@
+packages/
```

Comment:
Could just use one of the template gitignores for C# projects here but not fussed